step-templates/upgrade_to_latest_tentacle.json

```diff
@@ -0,0 +1,104 @@
+  },
+  "Parameters": [
+    {
+      "Id": "51ea8dde-8df6-4621-8c8a-f5cacf485c5e",
+      "Name": "OctopusUrl",
```

Comment:
Due to a potential conflict between project and step template variables (see https://github.com/OctopusDeploy/Issues/issues/2126), these parameters need to have a prefix to help ensure that they are unique. Typically the prefix is an abbreviated form of the step template's name e.g. `UpdateTentacles.OctopusUrl`.